src/agents/abmplot.jl

```diff
@@ -58,8 +58,11 @@ Requires `Agents`. See also [`abmvideo`](@ref) and [`abmexploration`](@ref).
   heatmap if `heatarray` is not nothing.
 * `static_preplot!` : A function `f(ax, model)` that plots something after the heatmap
   but before the agents.
-* `osmkwargs = NamedTuple()` : keywords directly passed to
-  `osmplot!` from OSMMakie.jl if model space is `OpenStreetMapSpace`.
+* `osmkwargs = NamedTuple()` : keywords directly passed to `OSMMakie.osmplot!` 
+  if model space is `OpenStreetMapSpace`.
+* `graphplotkwargs = NamedTuple()` : keywords directly passed to 
+  [`GraphMakie.graphplot!`](https://graph.makie.org/stable/#GraphMakie.graphplot) 
+  if model space is `GraphSpace`.
```

Comment:
This is fine. But also please add some documentation message in line 37 here, saying that for graphspace `ac, as, ac` are functions of _iterables_ of agents, e.g., if you want the size of a node in the graphplot to be the amount of agents you'd pass `as = length`.

Reply:
The docs explain it like [this](https://github.com/JuliaDynamics/InteractiveDynamics.jl/pull/114/files#diff-1e8d7ee69ea78766689a80a7e1d84ba021b126b86c9aaee8e16b666dc85cc0bbR68). I can add a similar thing to lines 19-21 in the docstring of the recipe.

Reply:
The docs explain it like [this](https://github.com/JuliaDynamics/InteractiveDynamics.jl/pull/114/files#diff-1e8d7ee69ea78766689a80a7e1d84ba021b126b86c9aaee8e16b666dc85cc0bbR68). I can add a similar thing to lines 19-21 in the docstring of the recipe.